package.json

```diff
@@ -9,7 +9,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e",
+    "e2e": "cypress run",
```

Comment:
I get the following when I try to run `npm run e2e`:

```
Cypress could not verify that this server is running:

  > http://localhost:4200

We are verifying this server because it has been configured as your `baseUrl`.

Cypress automatically waits until your server is accessible before running tests.

We will try connecting to it 3 more times...
We will try connecting to it 2 more times...
We will try connecting to it 1 more time...

Cypress failed to verify that your server is running.

Please start this server and then run Cypress again.
```

One way to resolve this is to change the instructions to run `npm run start` in a separate terminal window. A better option is to change the [`e2e` target's configuration in angular.json](https://github.com/trustthevote/MVP-App-TestHarness/blob/main/angular.json#L133-L147) to use Cypress instead of Protractor. The `devServerTarget` configuration there will ensure the dev server is properly spun up before running the tests. Let me know if that makes sense, and if you're comfortable making that change.

Reply:
I tried replacing protractor with cypress in angular.json, but I get the same error as above. I followed this guide: https://dev.to/christianmemije/adding-cypress-to-an-angular-project-30je putting the following into angular.json

```
"e2e": {
  "builder": "@nrwl/cypress:cypress",
  "options": {
    "devServerTarget": "app:serve",
    "cypressConfig": "./cypress.json",
    "tsConfig": "./tsconfig.json"
  },
  "configurations": {
    "production": {
      "devServerTarget": "app:serve:production"
    },
    "ci": {
      "devServerTarget": "app:serve:ci"
    }
  }
}
```

This doesn't seem to be enough the start the dev-server automatically when running e2e.

Reply:
No worries, thanks for taking a look at this one. I'll make a note to take a look at this later on.

Reply:
Update: got it working here! https://github.com/trustthevote/MVP-App-TestHarness/pull/15